### PR TITLE
Some error messages and spelling mistakes fixed related to operators

### PR DESCRIPTION
**Pull request by Neel-Shah-29**
This Pull request: fixes some error messages and spelling mistakes in operators 

## Checklist:

- [X] tested changes locally
- [ ] updated the docs (if necessary)

### CODE REVIEW

File: tmva/sofie/inc/TMVA/ROperator_BatchNormalization.hxx [hunk 1]
```diff
@@ -107,12 +107,12 @@ public:
 
 		fShapeX = model.GetTensorShape(fNX);
       
-      if (fShapeX.size() <  2 || fShapeX.size() > 4) {
-         throw
-				std::runtime_error("TMVA SOFIE BatchNormalization Op input tensor " + fNX + " fnx has wrong shape : " + ConvertShapeToString(fShapeX));
-      }
+		if (fShapeX.size() <  2 || fShapeX.size() > 4) {
```
Comment (lmoneta) [inline tmva/sofie/inc/TMVA/ROperator_BatchNormalization.hxx:110]: The indentation here should be only 3 spaces and no tabs. 
I think also above is not correct. Could you add this fix too, please ?

**Review by lmoneta — CHANGES_REQUESTED** [review]
Looks good, thank you for the spelling fix. 
I have just a small request on fixing the indentation to the ROperator_BatchNormalization.hxx file

**Comment by lmoneta** [comment]
You can look also at the result of clang-format that will be available after I trigger a build

File: tmva/sofie/inc/TMVA/ROperator_BatchNormalization.hxx (hunk 1 above)
Comment (Neel-Shah-29) [reply tmva/sofie/inc/TMVA/ROperator_BatchNormalization.hxx]: Yes Done.

**Review by lmoneta — APPROVED** [review]
LGTM

**Pull request merged**